recipe/meta.yaml

```diff
@@ -44,33 +45,34 @@ requirements:
     - python
     - pip
     - cython
-    - arrow-cpp >=8,<9
+    - arrow-cpp >=10,<11
```

Comment:
I would suggest to use `arrow-cpp 10.0.1` because it's in the range of `pyarrow` https://github.com/snowflakedb/snowflake-connector-python/blob/fbf5954c82a6db45a441c4c81dbf6d9b61a0b592/pyproject.toml#L9

Reply:
Good catch!